### PR TITLE
Add blindsigning in Settings for nano.

### PR DESCRIPTION
**Pull request by ajinkyaraj-23**
Options are "For Large Tx", "ON", "OFF"
Update snapshots for nano

### CODE REVIEW

**Review by spalmer25 — COMMENTED** [review]
Except the few remarks, it looks good to me.

**Comment by spalmer25** [comment]
The error you're getting from the generated tests is caused by the `set_expert_mode` function in the `test_runtime.sh` file because the settings flow has changed.
Once you have corrected this function, the tests should pass.

File: tests/integration/nano/snapshots/nanos/settings_max_screen_30.png
[the remark this answers is not in the document]
Comment (ajinkyaraj-23) [reply tests/integration/nano/snapshots/nanos/settings_max_screen_30.png:1]: Done.

File: app/src/ui_settings.c [hunk 1]
```diff
@@ -31,16 +31,31 @@ expert_mode_toggle()
...
+{
+    FUNC_ENTER();
+    toggle_blindsign_status();
+    ui_settings_init(SETTINGS_BLINDSIGN_PAGE);
+    FUNC_LEAVE();
+}
+
+UX_STEP_CB(ux_expert_mode_step, bn, expert_mode_toggle(), {"Expert mode",
+           global.expert_mode_state});
+UX_STEP_CB(ux_blindsign_step, bn, blindsign_toggle(), {"Allow Blinsigning",
```
[the remark this answers is not in the document]
Comment (ajinkyaraj-23) [reply app/src/ui_settings.c]: Done.

File: app/src/globals.h [hunk 2]
```diff
@@ -112,8 +123,8 @@ typedef struct {
     struct {
         bagl_element_t bagls[4 + TZ_SCREEN_LINES_11PX];
     } ux;  /// Config for history screens for nano devices.
-    char expert_mode_state[10];  /// Expert mode state to be displayed in
-                                 /// settings.ENAELED/DISABLED.
+    char expert_mode_state[10];  /// Expert mode text:  "ENAELED", "DISABLED"
+    char blindisign_state_desc[14];  /// Blindsigning text: "For Large Tx", "ON" , "OFF"
```
[the remark this answers is not in the document]
Comment (ajinkyaraj-23) [reply app/src/globals.h]: Done.

File: tests/integration/nano/snapshots/nanos/settings_max_screen_30.png
[the remark this answers is not in the document]
Comment (spalmer25) [reply tests/integration/nano/snapshots/nanos/settings_max_screen_30.png:1]: There are still some snapshots are unused. (These of NanoSP and NanoX)

**Pull request merged**